src/main/ngapp/src/app/chat-bar/chat-bar.component.ts

```diff
@@ -20,7 +22,19 @@ export class ChatBarComponent implements OnInit {
     this.currId = localUser.uid;
   }
 
-  onSelect(user: User) {
-    this.changeChat.emit(user);
+  getLastMessage(messages: Message[]): string {
+    let message: string;
+
+    if(this.currId === messages[messages.length - 1].senderId){
```

Comment:
message := messages[messges.length - 1]
return this.currID ==== message.SenderId ? message.text : message.translated;